### DISCUSSION TITLE
Assigning Link Objects (in pydantic models)

### DISCUSSION THREAD

**Discussion by Gizrah** [fastapi/fastapi#7630] [Questions]
### First check

* [x] I used the GitHub search to find a similar issue and didn't find it.
* [x] I searched the FastAPI documentation, with the integrated search.
* [x] I already searched in Google "How to X in FastAPI" and didn't find any information.

### Description

How can I create Link Objects (https://swagger.io/specification/#linkObject) properly for my models?
My basic User model can be posted with a List[int], of which the int value represents the ID value of a different model (Provider).

I'd like to access the (randomly generated) operationId for the Provider model's `get` endpoint, then put that in a `links` property in the User model. The OpenAPI spec's Link Object allows references to be made, but how do I do this in FastAPI? I can see that Field could do something similar, although I don't know if the end result is properly spec'ed.

**Comment by Gizrah** [comment]
I've found that apparently my reading comprehension was lacking, as the link object is defined in the response(s) objects, not the schema. Admittedly, it's the last word on the wiki about additional responses, so I just missed it.

**Comment by tiangolo** [comment]
Thanks for reporting back and closing the issue :+1:
